### DISCUSSION TITLE
What support does a maintainer need?

### DISCUSSION THREAD

**Discussion by nanorepublica** [carltongibson/notes#32] [Being an open source maintainer]
There are the few obvious categories:

* Time
* Money
* Others maintainers

But I'm interested in hearing about the smaller items that a maintainer might like to have support with or access to? Please get specific.

**Comment by vossisboss** [comment]
Education on best practices and what works well in the long run. Suggestions for automation tools, tests, and things that make would help make maintenance less of a chore.
